pyscriptjs/examples/hello_world.html

```diff
@@ -7,9 +7,9 @@
     <title>PyScript Hello World</title>
 
     <link rel="icon" type="image/png" href="favicon.png" />
-    <link rel="stylesheet" href="../build/pyscript.css" />
+    <link rel="stylesheet" href="build/pyscript.js" />
```

Comment:
I think this is a typo... `js --> css`

Reply:
Also `href="build/pyscript.js` -> `href="../build/pyscript.css`